workspace-manager-clienttests/src/main/resources/servers/workspace-resiliency.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "workspace-resiliency",
+  "description": "Workspace Manager Resiliency Environment",
+
+  "samUri": "https://sam.dsde-dev.broadinstitute.org",
+  "datarepoUri": "https://jade.datarepo-dev.broadinstitute.org/",
+  "workspaceManagerUri": "https://workspace.ichang.integ.envs.broadinstitute.org/",
+
+  "cluster": {
+    "clusterName": "gke_terra-kernel-k8s_us-central1-a_terra-integration",
+    "clusterShortName": "terra-integration",
+    "region": "us-central1",
+    "zone": "us-central1-a",
+    "project": "terra-kernel-k8s",
+    "namespace": "terra-ichang",
+    "containerName": "workspacemanager-deployment"
+  },
+  "deploymentScript": {},
+  "testRunnerServiceAccountFile": "testrunner-k8s-sa.json",
+
+  "skipDeployment": true,
+  "skipKubernetes": false
+}
```

Comment:
multiple hardcoding of "ichang" in  this definition

Reply:
As described in previous comments. Currently `terra-ichang` is the only namespace with the test runner RBAC setup to manipulate k8s resources. Test runner SAs and RBAC setup are to be automated through terraform in accordance with steps in the [Test Runner Service Accounts](https://docs.google.com/document/d/1-fGZqtwEUVRMmfeZfUVrn2V3M2D_eonNLepuX0ve6nM/edit?usp=sharing) document. However, if other namespace need the test runner setup for resiliency tests, it will be easy to do now with all the steps documented so don't hesitate to reach out to me if you need resiliency tests to target other namespaces.